nextflow-genomics-cluster-ubuntu/scripts/init.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/sh
+#!/bin/bash
+# $1 = Azure storage account name
+# $2 = Azure storage account key
+# $3 = Azure file share name
+# $4 = mountpoint path
+# $5 = should run as nf node
+# $6 = username of nextflow user
+# $7 = number of tasks assigned to each node (cluster.maxCpus)
+
+log () {
+    echo "-------------------------" | tee -a $2
+    date -Is | tee -a $2
+    echo $1 | tee -a $2
+    echo "-------------------------" | tee -a $2    
+}
+
+#Install CIFS and JQ (used by this script)
+log "Installing CIFS and JQ" /tmp/nfinstall.log 
+apt-get -y update | tee /tmp/nfinstall.log
+apt-get install cifs-utils sudo apt-transport-https wget -y | tee -a /tmp/nfinstall.log
+
+#Create azure share if it doesn't already exist
+log "Installing AzureCLI and Mounting Azure Files Share" /tmp/nfinstall.log 
+echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | \
+    sudo tee /etc/apt/sources.list.d/azure-cli.list 
+
+apt-key adv --keyserver packages.microsoft.com --recv-keys 417A0893 | tee -a /tmp/nfinstall.log
+apt-get -y update | tee /tmp/nfinstall.log
+apt-get install azure-cli -y | tee -a /tmp/nfinstall.log
+
+az storage share create --name $3 --quota 2048 --connection-string "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=$1;AccountKey=$2" | tee -a /tmp/nfinstall.log
+
+#Wait for the file share to be available. 
+sleep 10
+
+DATA_DIR="/datadisks/disk1"
+if ! [ -f "vm-disk-utils-0.1.sh" ]; 
+then
+    DOWNLOAD_SCRIPT="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/shared_scripts/ubuntu/vm-disk-utils-0.1.sh"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts

Reply:
Refactored to use artifact url and be passed onto the box through custom script extension rather than wget in init.sh